MANIFEST.in

```diff
@@ -1,5 +1,6 @@
 include LICENSE README.rst
 recursive-include tests *.py
-include examples/example*.py
+include examples/*.py
 include requirements.txt tests/requirements.txt
 include docs/*
+include assets/mechanical-soup-logo.png
```

Comment:
IIRC it was deliberate not to have the image in the distribution. I need to check the history 'cause I dont remember the ins and outs (will do ASAP).

Reply:
Oops, I totally forgot that, sorry for the late answer.

The issue I had in mind was #239, where we decided to link to the logo hosted here on PyPI to keep the source distribution small. I'm not opposed to changing this, but I'd like @hemberger 's opinion before we do.

The source distribution is currently ~50kB, and the logo is ~100kB so the difference is both small in terms of absolute size and huge in relative size.

A possible alternative to including the logo is to change the link to point directly to the GitHub version, although people who use the locally built doc probably prefer to also have the logo locally.

Reply:
My feeling is that it not important to this project to include the logo in the distribution, so I would be in favor of pointing directly to the GitHub version. This would also simplify writing the `long_description` in setup.py, since we wouldn't need to find/replace the relative path to the logo there either.

For what it's worth, Requests links to its images on GitHub (and doesn't include its docs in the manifest either).